src/Webroot.php

```diff
@@ -147,16 +147,20 @@ public function putassets($sspak) {
 		$assetsOldPath = $assetsPath . '.old';
 		$assetsParentArg = escapeshellarg(dirname($assetsPath));
 
+		// Remove assets.old
```

Comment:
Can you explain the reasoning behind moving this to before the restore? This is functional change and seems out of scope of the PR - and I'm happy to take a look at it separately, it seems to have little to do with fixing the issue where `$options` aren't passed through from `sudo`.

Otherwise it seems to me that the `mv` command will replace the `assets.old` dir, so we don't need to wipe it.